index.html

```diff
@@ -363,6 +375,10 @@
                         <dd>
                             <p>Indicates if the pointer represents the <a>primary pointer</a> of this pointer type.</p>
                         </dd>
+                    <dt><dfn>getCoalescedEvents</dfn></dt>
+                        <dd>
+                            <p>Returns a sequence of all <code>PointerEvents</code> that were coalesced into the event. The coalesced events will have the same value as the event in attributes like <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The event itself will have the greatest <a href="https://dom.spec.whatwg.org/#dom-event-timestamp">timeStamp</a> ([[!WHATWG-DOM]]) among the events in the sequence and the first event in the sequence will have the smallest timeStamp. None of the events in the returned sequence will have coalesced events. When an event has no coalesed event this function returns an empty sequence.</p>
```

Comment:
Suggested wording change from "The coalesced events will have the same value as the event in attributes like ..." to "The following attributes of the coalesced events will always have the same value as the event: ..."

I wonder if there's a better way to refer to the primary event object to make "the event" less confusing when used in the same sentence as "coalesced events."